environments/singularity-builder/README.md

```diff
@@ -0,0 +1,13 @@
+# Singularity Builder Environment
+
+Configuration of the environment to build planners and related software through singularity.
+
+## To run
+
+```bash
+curl -fsSL https://raw.githubusercontent.com/AI-Planning/planutils/master/environments/singularity-builder/setup.sh | sh
+```
+
+## Note
+
+If using inside of the `planutils` docker imagine, you must run it with `--privileged` mode.
```

Comment:
docker imagine => docker image

Reply:
But are you sure that's really a bug? ;)